package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -209,3 +218,155 @@ def __init__(self, strand1: List[Residue], strand2: List[Residue],
             sel2.append(s[1].atoms.select_atoms(f'name {a2}'))
 
         super(WatsonCrickDist, self).__init__(sel1, sel2, **kwargs)
+
+
+class MinorPairDist(NucPairDist):
+    r"""Minor-Pair basepair distance for selected residues over a trajectory.
+
+    Takes two lists of :class:`~MDAnalysis.core.groups.Residue` objects and
+    calculates the Watson-Crick distance between them over the trajectory.
+    Bases are matched by their index in the lists given as arguments.
+
+    Parameters
+    __________
+    strand1: List[Residue]
+        First list of bases
+    strand2: List[Residue]
+        Second list of bases
+    o2_name: str (optional)
+        Name of Oxygen 2 of nucleic acids
+        by default assigned to N1
+    c2_name: str (optional)
+        Name of Carbon 2 of nucleic acids
+        by default assigned to N3
+    g_name: str (optional)
+        Name of Guanine in topology
+        by default assigned to G
+    a_name: str (optional)
+        Name of Adenine in topology
+        by default assigned to G
+    u_name: str (optional)
+        Name of Uracil in topology
+        by default assigned to U
+    t_name: str (optional)
+        Name of Thymine in topology
+        by default assigned to T
+    c_name: str (optional)
+        Name of Cytosine in topology
+        by default assigned to C
```

Comment:
Would it be clearer to use `guanine="G"` instead of `g_name="G"` and same for everything else?

I find the meaning to be clearer when reading `guanine="G"`.

Reply:
I updated it to use the full name, ex: `guanine_name=G`

Reply:
Now I see how we got to the API-breaking naming... my comment from a year ago when it would have been easy to change the API without a break. As I said elsewhere, `g_name` etc work.